_stories/molecules/Table/README.md

```diff
@@ -0,0 +1,150 @@
+# Table
+
+The `<Table>` component creates a table from a provided data source. It allows for column configuration, alpha/numeric sorting, custom sorting, and theming.
+
+## Usage
+
+Simple usage of the `<Table>` just requires a `data` and `columns` prop. Data must be provided as an array of objects with keys that match column identifiers. Columns are also provided by an array that identifies the keys of the data to use. Array order will determine their order in the table. Here's an example:
+
+```js
+const data = [
+    {
+        name: 'John',
+        age: 36
+    },
+    {
+        name: 'Jane',
+        age: 32
+    }
+];
+
+const columns = ['name', 'age']; // table columns will be ordered by the array
+
+const People = () => <Table columns={columns} data={data} />;
+```
+
+## Props
+
+The following props can be passed to the `<Table>` component:
+
+| name       | type       | description                                                                                                                                                                                                                                                | default |
+| ---------- | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| data       | `Array`    | An `Array` of `Objects` with simple key/value pairs.                                                                                                                                                                                                       |         |
+| columns    | `Array`    | An `Array` of `Strings` identifying the keys from the `data` to be displayed in columns. [Customization of columns](#columns) can be made by providing an `Array` of `Objects`. Order of the columns in the table is determined by the order of the array. |         |
+| children   | `node`     | To render children in the `<Table>`. Use if you want to manually construct a table.                                                                                                                                                                        |         |
+| className  | `String`   | Additional classes to be added to the `<table>`.                                                                                                                                                                                                           |         |
+| context    | `String`   | Theme context of the table. One of `secondary`.                                                                                                                                                                                                            |         |
```

Comment:
Are there any other contexts besides secondary?

Reply:
Nope. I thought about changing it to a boolean, like `isSecondary`,  which might be better? Originally I thought there were more contexts like some of the other gumdrops which is why it is the way it currently is. :)